src/domain/checklist/preLoadChecklist.ts

```diff
@@ -6,8 +6,8 @@ import { PreLoadRecord } from '../PreLoadRepository';
 import { Checklist } from './types';
 
 export const check0 = {
-  name: 'Exists a response with status code 401',
-  eval: RA.some((record: PreLoadRecord) => record.output.statusCode === 401),
+  name: 'Exists a response with status code 403',
+  eval: RA.some((record: PreLoadRecord) => record.output.statusCode === 403),
```

Comment:
I have some doubts on this, because it looks like the 403 is the status returned in case of invalid API key, but from what we have write [here](https://pagopa.atlassian.net/wiki/spaces/VL/pages/525238617/The+endpoints+to+check#Request-an-%E2%80%9Cupload-slot%E2%80%9D), after a meeting with PN's team, it looks like we should check that there is a response with 401 status code